Session/Storage/Handler/RedisSessionHandler.php

```diff
@@ -194,7 +193,7 @@ protected function lockSession($sessionId)
         $attempts = (1000000 / $this->spinLockWait) * $this->lockMaxWait;
 
         $this->token = uniqid();
-        $this->lockKey = $sessionId.'.lock';
+        $this->lockKey = $this->getRedisKey($sessionId).'.lock';
```

Comment:
@Seldaek in fact this line make sure the prefix is also applied to the lock. I'll add some test.
Is there anything else?

Reply:
I see that but before getRedisKey() was applied to every usage of $this->lockKey so I don't see how it changes anything.

Reply:
Got it now! Thx!